lib/server.js

```diff
@@ -177,7 +177,7 @@ class Server extends KarmaEventEmitter {
       }
 
       webServer.listen(this._boundServer, () => {
-        this.log.info(`Karma v${constant.VERSION} server started at ${config.protocol}//${config.listenAddress}:${config.port}${config.urlRoot}`)
+        this.log.info(`Karma v${constant.VERSION} server started at ${config.protocol}//${config.hostname}:${config.port}${config.urlRoot}`)
```

Comment:
If you don't mind, could you please explain the difference between `listenAddress` and `hostname` in the commit message? I have no idea what the distinction is..

Reply:
done